test/library_server.cpp

```diff
@@ -115,6 +115,7 @@ std::string maskVariableOPDSFeedData(std::string s)
 )
 
 #define CHARLES_RAY_CATALOG_ENTRY _CHARLES_RAY_CATALOG_ENTRY("zimfile%26other")
+#define NO_MAPPER_CHARLES_RAY_CATALOG_ENTRY _CHARLES_RAY_CATALOG_ENTRY("charlesray")
```

Comment:
Proper alignment will help to emphasize the similarities and differences between these two lines. It would be even better if `NO_MAPPER_CHARLES_RAY_CATALOG_ENTRY` is renamed to `CHARLES_RAY_CATALOG_ENTRY_NO_MAPPER`. The same applies to `NO_MAPPER_RAY_CHARLES_CATALOG_ENTRY`

Reply:
Done

Reply:
The suggestion to rename was meant to be *in addition* to the suggestion to align:

```c++
#define CHARLES_RAY_CATALOG_ENTRY           _CHARLES_RAY_CATALOG_ENTRY("zimfile%26other")
#define CHARLES_RAY_CATALOG_ENTRY_NO_MAPPER _CHARLES_RAY_CATALOG_ENTRY("charlesray")
```